packages/vscode/__fixtures__/js-glimmerx-app/src/SimpleComponent.js

```diff
@@ -0,0 +1,15 @@
+import Component from '@glint/environment-glimmerx/component';
+import { hbs } from '@glimmerx/component';
+
+/**
+ * @typedef SimpleComponentArgs
+ * @property {object} Args
+ * @property {string} Args.message
+ */
+
+/** @extends {Component<SimpleComponentArgs>} */
```

Comment:
Generally the terminology we've been encouraging is `Signature` to encompass the thing that contains `Args`/`Yields`/etc.

```suggestion
/**
 * @typedef SimpleComponentSignature
 * @property {object} Args
 * @property {string} Args.message
 */

/** @extends {Component<SimpleComponentSignature>} */
```